### DISCUSSION TITLE
v0.1.3

### DISCUSSION THREAD

**Discussion by ryrobes** [ryrobes/rvbbit#5] [Announcements]
- Fixed (bandaged?) JS memory leak created in v0.1.1
- Data Board will go into a partial "display mode" when no mouse activity is detected for 60 seconds. This is similar to the effect you get when you hit the display mode toggle on the bottom right. Remove some UI clutter, less to render, etc.
   - Good for displaying boards for long periods of time, looks nice and clean.
- Added an optional (advanced user) :root-mod key to be in the base card map. This allows Cards to stick to the sides, top or bottom - but applying specific DSL code to the x,y coords. For example you can set :root-mod [[:right -4] [:bottom -5]] and depending on the size of your card (x,y is the top left corner) this would stick the card to the bottom right, no matter what the size of the browser is. :top, :bottom, :left, :right, :middle are all available.
    - to see an example of this - there is now a server-clock screen that tries to keep the clock cards on bottom middle regardless of the size and orientation of the screen. Dynamic size is likely next, but that is slightly trickier with diff use cases. In most scenarios I think that fixed layouts are best. Perhaps it is better to take the Tableau route and have pseudo screen-sizes and each is just a reconfigutation of the coords and dimensions of the content from a "base" screen config
- specialized data types are now strigified when they head to the grid UI to prevent "tag reader" errors - however, if you want to manipulate the native objects from the query - you should be able to access them in a repl via :data/view-name (more on this later!) - fixes #4 hopefully (tested it on every "odd" data type I could find on Postgres).

**Full Changelog**: https://github.com/ryrobes/rvbbit/compare/v0.1.1...v0.1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/ryrobes/rvbbit/releases/tag/v0.1.3'>v0.1.3</a>.</em>

**Comment by tombarys** [comment]
Dear Ryan, I am sorry for being dummy here🙈, but I have not found info about installation. I would like to play with alpha on MacOS. Is that even possible? Thanks!
